### PR TITLE
license file

### PR DESCRIPTION
**Pull request by karlthepagan**
People who need to sign off-

- [x] Robert Hafner 
- [x] Nolan Cafferky
- [x] Clarence "Sparr" Risher
- [x] "semperrabbit"
- [x] Nhan Ho
- [x] James Cook
- [x] David Ross
- [ ] Radu Creosteanu
- [ ] "RueMic"
- [ ] "n00bish" 
- [ ] Remco Overdijk,
- [ ] "Salmakis"
- [ ] seancl
- [ ] Zoey "Dessix

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,23 @@
+MIT License
+
+Copyright (c) 2016 Robert Hafner, Clarence Risher, "semperrabbit", Nhan Ho,
+"Esryok", David Ross, Radu Creosteanu, "RueMic", "n00bish", Remco Overdijk,
+"Salmakis", seancl, and Zoey "Dessix".
```
Comment (tedivm) [inline LICENSE]: Missing members of original repository. Additionally each person will need to sign off in this pull request before this will be merged.

Comment (semperrabbit) [inline LICENSE]: I concur with the verbiage in this PR.

Comment (karlthepagan) [inline LICENSE]: This includes the contributors list from https://github.com/LeagueOfAutomatedNations/loan_website

I need more specific feedback to proceed.

Comment (tedivm) [inline LICENSE]: The "contributors" lists are never complete on Github. Anyone who doesn't have git setup with the exact email address that github has on file isn't shown in that list.

Comment (karlthepagan) [inline LICENSE]: Thanks, I'll check the repo itself.

Comment (tedivm) [inline LICENSE]: Make sure to go through both.

Comment (nolanfu) [inline LICENSE]: +1 to adding MIT license.

Comment (tedivm) [inline LICENSE]: @nolanfu - can you comment in the main conversation, not this comment? The comment will disappear when the code changes.

**Comment by karlthepagan** [comment]
@nolanfu indeed you were the one missing

**Comment by karlthepagan** [comment]
@sparr @NhanHo @Esryok @daboross please comment if you approve or reject distributing your code under the MIT license

@semperrabbit indicated his consent in the line review here: https://github.com/LeagueOfAutomatedNations/Screeps-LoAN/pull/98/files/e8c1e0f6b988c4ba536d990f66e76fda7e937e86

**Comment by sparr** [comment]
approve, prefer credit as `Clarence "Sparr" Risher`

**Comment by Esryok** [comment]
I'm on board, name is James Cook.

**Comment by daboross** [comment]
I'm alright with MIT.

**Comment by NhanHo** [comment]
Approve, name is Nhan Ho.

**Comment by sparr** [comment]
@karlthepagan @tedivm that's everyone, I think

**Comment by tedivm** [comment]
There are fourteen names on the copyright document. There are six people who have signed off on this thread (seven if you count me, as I apparently I hadn't done so above, but to be clear I sign off).

Look, this is the second time you've tried to prematurely get this accepted. This isn't a joking matter- it's a legal one. I don't see the point in merging an invalid license in- once everyone signs off we will merge it, but not a moment sooner. Otherwise we face the horrible potential of people thinking this is open source, using it, and then having someone else object down the line (who knows, maybe someone will refuse MIT license and push for GPL or something).

**Comment by sparr** [comment]
Not sure who the "you" in question is, but it wasn't me the first time, and I'm not "trying to get this accepted" now. I haven't been keeping track of all the requirements here. If other people need to be called in, great. I was just trying to bring your/karl's attention to the fact that everyone who was called has come.

**Comment by tedivm** [comment]
@sparr - my apologies, it looked like your comment was coming from Karl. He's been rather antagonistic towards me so I've been on edge when dealing with him.

**Comment by karlthepagan** [comment]
IANAL but n00bish, seancl, Salmakis, Dissix, RueMic, Remco, Radu are all authors who have not consented in this PR but their contributions would be under the terms of the MIT license as present beginning Sep 28, 2016

The only contributor to the old codebase which might fall out of scope is Tedivm.

**Comment by tedivm** [comment]
Alright, I'll review this evening. As long as no one is missing it will get merged.

**Comment by sparr** [comment]
Clarify for me: what happened on Sep 28 2016? I don't see a license file in the repo as of today.

**Comment by karlthepagan** [comment]
@sparr here's that commit

**Comment by sparr** [comment]
Ahh, ok. I'm unfamiliar with the relationship between the deprecated repo and this one, and the contributors to each. I leave it to you guys to figure out. Thanks for the link.

**Comment by Creosteanu** [comment]
I don't remember making modifications to loan, but I approve of the license.

**Pull request merged**